classes/affiliation/Affiliation.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/**
+ * @file classes/affiliation/Affiliation.php
+ *
+ * Copyright (c) 2024 Simon Fraser University
+ * Copyright (c) 2024 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class Affiliation
+ *
+ * @ingroup affiliation
+ *
+ * @see DAO
+ *
+ * @brief Basic class describing an affiliation.
+ */
+
+namespace PKP\affiliation;
+
+use PKP\core\DataObject;
+
+class Affiliation extends DataObject
+{
+    /**
+     * Get author id
+     */
+    public function getAuthorId()
+    {
+        return $this->getData('authorId');
+    }
+
+    /**
+     * Set author id
+     */
+    public function setAuthorId($authorId): void
+    {
+        $this->setData('authorId', $authorId);
+    }
+
+    /**
+     * Get the ROR
+     */
+    public function getROR(): ?string
+    {
+        return $this->getData('ror');
+    }
+
+    /**
+     * Set the ROR
+     */
+    public function setROR($ror): void
+    {
+        $this->setData('ror', $ror);
+    }
+
+    /**
+     * Get name
+     */
+    public function getName(): ?array
+    {
+        return $this->getData('name');
+    }
+
+    /**
+     * Set name
+     */
+    public function setName(?array $name): void
```

Comment:
I think we should stay consistent with other 'settings' method that could have locale and have something like:
```
    public function setName(string|array $name, ?string $locale = null)
    {
        $this->setData('name', $name, $locale);
    }

    public function getName(?string $locale = null)
    {
        return $this->getData('name', $locale);
    }
```
When you then provide an array as $name and no locale to setName() it will/should be set correctly.